unturned/OpenMod.Unturned/Players/Events/Chat/UnturnedServerSendingMessageEvent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using SDG.Unturned;
+using UnityEngine;
+using Event = OpenMod.Core.Eventing.Event;
+
+namespace OpenMod.Unturned.Players.Events.Chat
+{
+    public class UnturnedServerSendingMessageEvent : Event
```

Comment:
This event might be unnecessary

Reply:
I am more interested in what use cases this would cover, otherwise the naming is fine

Reply:
Sounds good then